src/st/cutscene_unk3.h

```diff
@@ -4,17 +4,17 @@
 #endif
 
 #if defined(VERSION_PSP)
-extern void* D_pspeu_0927B6A8;
+extern s32 cutsceneUnk3Unknown[0x380]; // bss
 #endif
 
-static void CutsceneUnk3(s16 arg0) {
+static void CutsceneUnk3(s16 nextCharY) {
```

Comment:
Renamed based on usage, this is called from `CutsceneUnk4` with `g_Dialogue.nextCharY` so seemed sensible to renam